utils/utils_test.go

```diff
@@ -1,18 +1,48 @@
 package utils
 
 import (
-	"os"
+	"fmt"
+	"github.com/attestantio/go-eth2-client/spec/bellatrix"
+	"github.com/attestantio/go-eth2-client/spec/electra"
+	"github.com/golang/snappy"
+	"io/ioutil"
+	"net/http"
 	"testing"
 
 	"github.com/attestantio/go-builder-client/api"
 	"github.com/attestantio/go-eth2-client/spec"
-	"github.com/attestantio/go-eth2-client/spec/bellatrix"
 	"github.com/attestantio/go-eth2-client/spec/capella"
 	"github.com/attestantio/go-eth2-client/spec/deneb"
-	"github.com/attestantio/go-eth2-client/spec/phase0"
 	"github.com/stretchr/testify/require"
 )
 
+const mainnetTests = "https://github.com/ethereum/consensus-spec-tests/raw/master/tests/mainnet/"
```

Comment:
is there a way to make tests run locally without downloading anything? maybe as a submodule or have the spec tests as part of the repo

Reply:
Yes, sorry I meant to do that. This was a hacky way of retrieving arbitrary files. I thought I might need `pre.ssz_snappy` or `post.ssz_snappy` too.

Speaking of which, there's a problem with the block hash in the Electra reference tests. It does not match the value that ELs actually produce. It's because the CL specs introduce two fields to the block header, when ELs have decided to only add a single "requests" fields. We need to fix the specs first.